client/src/App.vue

```diff
@@ -41,9 +43,15 @@ export default defineComponent({
       state,
     };
   },
+  watch: {
+    $route(to: RouteLocation) {
+      // TODO: consider adding a string that says this is a non-prod environment, so devs can see
```

Comment:
How come?

Reply:
I'll have tabs open for dev and local and sandbox environments and this would make it easier to distinguish.

Reply:
SGTM